cli/simulate_pixels.py

```diff
@@ -221,30 +242,42 @@ def run_simulation(input_filename,
     _, _, rev_idx = np.intersect1d(tot_evids, tracks['eventID'][::-1], return_indices=True)
     end_idx = len(tracks['eventID']) - 1 - rev_idx
 
+    # create a lookup table for event timestamps
+    event_times = fee.gen_event_times(tot_evids.max()+1, 0)
+
     # We divide the sample in portions that can be processed by the GPU
     step = 1
+    
+    # charge simulation
+    event_id_list = []
```

Comment:
Can you clarify the logic here? I had moved these inside the for in order to save the output after each event, does it now happen every `EVENT_BATCH_SIZE` events?

Reply:
That's right. It helps speed up the Module 0 simulation a bit, since each event isn't actually that big. That can be adjusted to optimize for different detectors (or made into a script parameter). 

As a side note, I did a little bit of profiling and it now spends about equal time in the "simulation" piece as in the "export_to_hdf5" piece. So, there's a potential x2 speed-up that could be achieved by letting the export to hdf5 run in a separate process. Just something to think about - it would take a fair bit of work.